### PR TITLE
first

### PR DESCRIPTION
**Pull request by TMTIME**
thats it rebase complete

### CODE REVIEW

**Comment by Qustinnus** [comment]
@TMTIME is the dme the TG.dme copied and renamed or did you get our old one? (hint: the first one is what we need)

**Comment by Qustinnus** [comment]
gonna test something dont mind this

File: _FTL/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+#keep dirty tg cpde from touching our high qualitly and lag less ftl code
```
Comment (Qustinnus) [inline _FTL/README.md]: cpde lmao

**Comment by TMTIME** [comment]
copy pasted

**Pull request merged**